FakebookNotifications/FakebookNotifications.WebApi/Controller/NotificationController.cs

```diff
@@ -34,7 +34,8 @@ public NotificationController(IHubContext<NotificationHub> _hub, INotificationsR
         [HttpPost("comment")]
         public async Task<IActionResult> CommentNotificationAsync(string loggedInUser, string triggerUser, int postId)
         {
-            return await CreateNotificationAsync(loggedInUser, triggerUser, postId, "comment");
+            string notificationType = "comment";
```

Comment:
I meant declare the string "comment" once, and reference that one location both from methods like this and from the CreateNotificationAsync method. that will let the compiler catch typos in the notification type. i guess an enum would be better - you can make it a private nested enum